esp/services/ldapenvironment/ldapenvironmentService.cpp

```diff
@@ -125,7 +120,7 @@ bool CldapenvironmentEx::changePermissions(const char * ou, const char * userFQD
     action.m_action = "update";
     action.m_basedn = baseDN;
     action.m_rname =  rName;
-    action.m_rtype = RT_DEFAULT;
+    action.m_rtype = RT_FILE_SCOPE;
```

Comment:
Is this method always for file scope resource type?

Reply:
Yes, and this method is only called from one place (to set perms for the LDAP Admin user). Previous to this fix, because of the RT_DEFAULT type, the code was building the fully qualified name with the incorrect prefix (CN= instead of OU=), so the call to change perms failed in AD